<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplinePanel</class>
 <widget class="QWidget" name="SplinePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>24</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string notr="true">Spline tool</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="icon">
     <property name="pixmap">
      <pixmap resource="../../../../../res/WorldEditor.qrc">:/Style/styles/dark/icons/spline_point.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelX">
     <property name="text">
      <string notr="true">X:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="xEdit">
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelY">
     <property name="text">
      <string notr="true">Y:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="yEdit">
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelZ">
     <property name="text">
      <string notr="true">Z:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="zEdit">
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="breakPoint">
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="icon">
      <iconset resource="../../../../../res/WorldEditor.qrc">
       <normaloff>:/Style/styles/dark/icons/point_break.png</normaloff>:/Style/styles/dark/icons/point_break.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="addPoint">
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="icon">
      <iconset resource="../../../../../res/WorldEditor.qrc">
       <normaloff>:/Style/styles/dark/icons/point_add.png</normaloff>:/Style/styles/dark/icons/point_add.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="deletePoint">
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="icon">
      <iconset resource="../../../../../res/WorldEditor.qrc">
       <normaloff>:/Style/styles/dark/icons/point_remove.png</normaloff>:/Style/styles/dark/icons/point_remove.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../res/WorldEditor.qrc"/>
  <include location="../../../../../res/WorldEditor.qrc"/>
 </resources>
 <connections/>
</ui>
